src/fastoad/notebooks/02_explanation/04_OpenMDAO.ipynb

```diff
@@ -0,0 +1,892 @@
+{
```

Comment:
<blockquote>There are 3 types of components, according to OpenMDAO documentation</blockquote><p>I would rather say "3 <strong>main</strong> types".</p><p>Also, I would feel more comfortable with "OpenMDAO provides 3 main types of components"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fast-aircraft-design/FAST-OAD/pull/426/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>